cmd/link/userrole/cmd.go

```diff
@@ -83,6 +91,11 @@ func run(cmd *cobra.Command, argv []string) (err error) {
 	if err != nil {
 		reporter.Errorf("Error getting current account: %v", err)
 	}
+	if args.accountID != "" && currentAccount.ID() != args.accountID {
+		reporter.Errorf("Invalid account ID '%s'. "+
+			"It doesnt match with the user session '%s'.", args.accountID, currentAccount.ID())
+		return err
+	}
```

Comment:
This is not the correct validation here. An org admin will be adding the label on behalf of the user, so they are not logged in as them. We need to use the `args.accountID` to set the label on that account, rather than the current session.